docs/performance-tuning.asciidoc

```diff
@@ -175,3 +175,17 @@ Limiting the number of spans that may be recorded will reduce memory usage.
 
 Reducing max spans could result in loss of useful data about what occurred within a request,
 if it is set too low.
+
+An alternative to limiting the maximum number of spans can be to drop spans with a very short duration, as those might not be that relevant.
+
+NOTE: Using a span filter does not reduce the load of recording the spans in your application, but merely filters them out before sending them to the APM Server.
+This, however, both reduces the amount of storage needed to store the spans in Elasticsearch, and the bandwidth needed to transport the data to the APM Server from the instrumented application.
```

Comment:
Thanks! We need a space here for the admonition block to render correctly. I'll add it in.

```suggestion

This, however, both reduces the amount of storage needed to store the spans in Elasticsearch, and the bandwidth needed to transport the data to the APM Server from the instrumented application.
```